Tests/check_avif_leaks.py

```diff
@@ -0,0 +1,42 @@
+from io import BytesIO
+
+import pytest
+
+from PIL import Image
+
+from .helper import is_win32, skip_unless_feature
+
+# Limits for testing the leak
+mem_limit = 1024 * 1048576
+stack_size = 8 * 1048576
+iterations = int((mem_limit / stack_size) * 2)
+test_file = "Tests/images/avif/hopper.avif"
+
+pytestmark = [
+    pytest.mark.skipif(is_win32(), reason="requires Unix or macOS"),
+    skip_unless_feature("avif"),
+]
+
+
+def test_leak_load():
+    from resource import RLIMIT_AS, RLIMIT_STACK, setrlimit
```

Comment:
Why are you importing this inside a function, and that twice (l. 22, l. 32)? I'd rather move this import to the top...

Reply:
That's a good question. This was copied from the leak test for jpeg2000 [here](https://github.com/python-pillow/Pillow/blob/95cff6e959bb3c37848158ed2145d49d49806a31/Tests/check_j2k_leaks.py#L22). I suspect this was done because those imports will fail on windows (judging by the skip pytest mark above it). But those could be handled at the top of the file with an `except ImportError`